src/pybind/mgr/utils/more_itertools.py

```diff
@@ -0,0 +1,7 @@
+from itertools import tee
```

Comment:
@callithea @Devp00l this is a new module. if we want to use it at runtime. we need to package it. but currently, neither ceph.spec.in nor debian/* packages this module.

i'd suggest promote this module and move it into `python-ceph-common` package.

@sebastian-philipp what do you think?

Reply:
@tchaikov thanks for your feedback!

Reply:
+1